### PR TITLE
MRG: Add option for SVG in report

### PR DESCRIPTION
**Pull request by Eric89GXL**
There is also `add_figs_to_section` with default `format='png'` that we could change to `format='svg'` by deprecation. I think it's a nicer default for most figures.

I didn't change `Epochs` plotting because it seems like it will be too heavy, however.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Okay made `svg` optional. There is a class default specified during construction that defaults to png (same behavior as `master`). It can be used with `parse_folder`, and when combined with the regex, should be sufficiently powerful for people.

The PNG version is 56 MB and takes 93 sec to produce on my machine. The SVG version is 77 MB and takes 98 sec to produce. The report for sample data is laggier when browsing, but could be made nicer by choosing which things to render in SVG.

Ready for review/merge from my end.

File: .gitignore [hunk 1]
```diff
@@ -55,6 +55,7 @@ tags
 doc/coverages
 doc/samples
 cover
+examples/visualization/foobar.html
```
Comment (agramfort) [inline .gitignore]: ???

File: examples/visualization/make_report.py [hunk 2]
```diff
@@ -6,32 +6,47 @@
...
+    figs.append(evoked.plot_topomap(t, vmin=-300, vmax=300, res=100,
+                                    show=False))
     plt.close(figs[-1])
-report.add_slider_to_section(figs, times, 'Evoked Response')
+report.add_slider_to_section(figs, times, 'Evoked Response',
+                             image_format='svg')
 
 # # to save report
-# report.save('foobar.html', True)
+report.save('foobar.html', True)
```
Comment (agramfort) [inline examples/visualization/make_report.py]: please comment out

**Comment by agramfort** [comment]
also please expose this option on the mne command line tool

File: examples/visualization/make_report.py (hunk 2 above)
Comment (Eric89GXL) [inline examples/visualization/make_report.py]: this comment answers your other comment :)

**Comment by Eric89GXL** [comment]
@agramfort done

**Comment by agramfort** [comment]
please merge when CIs are happy

thx @Eric89GXL

**Pull request merged**